components/Setter.jsx

```diff
@@ -0,0 +1,10 @@
+import styles from "@/styles/components/setter.module.scss";
+
+export default function Setter({ header, changer }) {
```

Comment:
This component is not a good idea.
1. It misleads the actual purpose of the component - not to set something but to layout (UI wise) the "setter" in a certain way.
2. It obfuscates simple the behaviour and structure of the layout (It will be more coherent to look at the settings page and see directly the structure).
3. What is the purpose of `header`? Is it a header? not really, it's just the name of the setter.
4. `changerElement` would be a better name to indicate  that it is a element/component you are passing, but regardless it's not a good idea to pass the setter that way (combined with the whole purpose of the component).

The bottom line is that not all behaviour that is repeating should be split/encapsulated.

Reply:
noted